client/app/dedicatedCloud/datacenter/drp/dedicatedCloud-datacenter-drp.constants.js

```diff
@@ -45,13 +54,22 @@ export const DEDICATEDCLOUD_DATACENTER_PCC_UNAVAILABLE_CODES = [
   403, 460,
 ];
 
+export const DEDICATEDCLOUD_DATACENTER_DRP_VPN_CONFIGURATION_STATUS = {
+  configured: 'configured',
+  configuring: 'configuring',
+  notConfigured: 'notConfigured',
+};
```

Comment:
Not much transformation is done

Reply:
It helps you write code this like:
`if (status ==== DEDICATEDCLOUD_DATACENTER_DRP_VPN_CONFIGURATION_STATUS .configured )`

instead of

`if (status ==== 'configured' )`

This way, the day `configured` is changed to `isConfigured`, you don't have to change all `'configured'` of the repository :)